js/main.js

```diff
@@ -35,6 +36,38 @@ function inflate_b64(base64) {
 	return JSON.parse(decompressed);
 }
 
+function update_sig_status(text, isError = false) {
+	const sigstatus = document.getElementById('sig-status');
+	sigstatus.innerHTML = text;
+	sigstatus.classList = isError ? "alert alert-error" : "alert alert-info";
+}
+
+function verify_signature(header, raw, xhr) {
```

Comment:
We use raw here, but below it's called token. Is token a better name for the argument?